### PR TITLE
Fixed issues

### PR DESCRIPTION
**Pull request by nishasen**
Fixed some of the issues as suggested after reviewing the app.
1. Fixed category card - removed emphasis
2. added toast in create playlist and fixed div
3. Fixed the side nav - added background color and Link to text
4. Fixed the post operation of history, added conditional, api request

### CODE REVIEW

**Review by coldpigli — APPROVED** [review]
Everything looks good. No changes required

File: vibes/src/Components/CategoryCard/CategoryCard.module.css [hunk 1]
```diff
@@ -50,25 +65,20 @@
...
-}
-
 .category_card:hover {
-    transform: scale(1.1);
+    transform: scale(1.02);
     transition: transform .3s;
     border: 2px solid var(--primary-hover);
 }
 
 .category_card:hover .category_desc {
```
Comment (vaibhav-mahalle) [inline vibes/src/Components/CategoryCard/CategoryCard.module.css:74]: learnt something new about the animations

File: vibes/src/Components/CreatePlaylist/CreatePlaylist.jsx [hunk 2]
```diff
@@ -14,7 +14,12 @@ const CreatePlaylist = ({onPlaylist}) => {
   const handleSubmit = e => {
     e.preventDefault();
     const playlist = {title: newPlaylist, description: "New playlist"};
-    usePostPlaylistAxios('playlists', playlist, videoDispatch, "POST_PLAYLISTS")
+    if(userLogin) {
```
Comment (vaibhav-mahalle) [inline vibes/src/Components/CreatePlaylist/CreatePlaylist.jsx]: you could have used ternary operator over here.

**Review by vaibhav-mahalle — APPROVED** [review]
added a minor suggestion please look into it.

**Pull request merged**